content/momentum/changelogs/M4ChangeLog.md

```diff
@@ -19,6 +21,219 @@
   <tr><td><a href="#4.0.0">Momentum 4.0.0 released on 2014-04-30</a></td></tr>
 </table>
 
+<h2><a name="4.4.0">Momentum 4.4.0 released on 2021-08-?? (Depending on installation type, all changes may not be applicable)</a></h2>
+<ul>
+<li>Feature: SD-1427  -  Bump versions of all packages required for ecconfigd/eccf </li>
+<li>Feature: SD-321  -  Upgrade to a newer curl version (using one from 2013) </li>
+<li>Fix: TR-2081  -  ectop should not throw errors on exited threads </li>
+<li>Fix: SD-1414  -  SMTP AUTH module does not support temporary auth failures </li>
+<li>Fix: SD-1417  -  fcm module crashes when given invalid mime </li>
+<li>Fix: SD-1426  -  messages could stuck in active queue forever when sent on a binding failed to connect to remote host </li>
+<li>Feature: SD-1438  -  smtp tracer enhanced outbound connection logging </li>
+<li>Feature: SD-1436  -  esmtp trace connection acceptance and destruction </li>
+<li>Feature: SD-1469  -  Bump json-c and sqlite versions and fix packaging </li>
+<li>Feature: SD-1472  -  smtp tracer enhanced outbound connection logging v2 </li>
+<li>Feature: SD-1479  -  smtp tracer enhanced outbound connection logging v3 </li>
+<li>Fix: SD-1437  -  esmtp trace gnutls handshake failure details </li>
+<li>Fix: SD-1276  -  permfail emails to NXDOMAIN immediately instead of tempfail as now in Sparkpost </li>
+<li>Fix: SD-1489  -  generic_delivery module overwrites smtp_state_notconnected state id </li>
+<li>Fix: SD-1453  -  consecutive outbound connection failures did not trigger message delay as expected </li>
+<li>Fix: SD-1457  -  Always use latest system timestamps in paniclog output </li>
+<li>Feature: SD-1395  -  log tls cipher used for SMTP injections and deliveries </li>
+<li>Feature: TR-2207  -  Add support for pthread rwlocks to Lua </li>
+<li>Fix: TR-2239  -  Reduce pinned Lua garbage by disabling pinning of Lua threads for completions </li>
+<li>Fix: SD-1494  -  Debug messages logged at critical  </li>
+<li>Feature: SD-1544  -  Provide an option to set msg:routing_domain without using context variables </li>
+<li>Fix: SD-1573  -  rcpt_type is not included in event data when domain is mixed case using SMTP API </li>
+<li>Feature: SD-1439  -  control command to list binding domain stats </li>
+<li>Fix: SD-1609  -  memory leak on Event when connection is closed due to throttle </li>
+<li>Feature: SD-1343  -  Enable periodic GC on Lua threads for SMTP reception sessions </li>
+<li>Feature: SD-1326  -  Upgrade jlog library to latest release </li>
```

Comment:
There are 2 tickets around this.  SD-2029 is mentioned below.  Can we just remove this SD-1326 mention?